.github/workflows/test.yml

```diff
@@ -3,14 +3,8 @@
 
 name: Tests
 on:
-  push:
-    branches:
-    - main
-  pull_request:
-    branches:
-    - 'main'
-    - 'feature/**'
-    - 'version-3.x/**'
```

Comment:
can you elaborate, why you've removed this?

Reply:
I wanted to make sure that my branch is ok before creating a pull request. I can revert this.

Reply:
Do I have to revert this? @dmitry-mukhin

Reply:
done